lang/de.xml

```diff
@@ -1,20 +1,265 @@
 <?xml version="1.0" encoding="utf-8"?>
 
 <lang>
+	<main_menubar>
+		<file name="Datei">
+			<open_pref_path>Öffne Präferenzen-Pfad...</open_pref_path>
+			<install_firmware>Installiere Firmware</install_firmware>
+			<install_pkg>Installiere .pkg</install_pkg>
+			<install_zip>Installiere .zip, .vpk</install_zip>
+			<install_license>Installiere Lizenz</install_license>
+		</file>
+		<emulation name="Emulation">
+			<last_apps_used>Zuletzt benutzte Anwendungen</last_apps_used>
+		</emulation>
+		<configuration name="Konfiguration">
+			<settings>Einstellungen</settings>
+			<user_management>Benutzer</user_management>
+		</configuration>
+		<controls name="Steuerung">
+			<keyboard_controls>Tastatur</keyboard_controls>
+			<controllers>Controller</controllers>
+		</controls>
+		<help name="Hilfe">
+			<about>Über</about>
+			<welcome>Willkommen</welcome>
+		</help>
+	</main_menubar>
+
+	<app_context>
+		<update_history>Update-Verlauf</update_history>
+		<information>Information</information>
+		<app_delete>Diese Anwendung und alle damit verbundenen Daten, wie Nutzerdaten, werden gelöscht.</app_delete>
+		<save_delete>Möchtest du die diese Nutzerdaten löschen?</save_delete>
+		<eligible>berechtigt</eligible>
+		<ineligible>nicht berechtigt</ineligible>
+		<level>Level</level>
```

Comment:
the word "Level" is "Kindersicherung" in german.

Reply:
```suggestion
		<level>Stufe</level>
```
If it's parental controls related, should be "Stufe" then